requirements/runtime.in

```diff
@@ -1,5 +1,10 @@
 twine ~= 6.0
 
+# NOTE: 1.12.0 and later enable support for metadata 2.4
+# NOTE: This can be dropped once twine stops using pkginfo
+# Ref: https://github.com/pypa/twine/pull/1180
+pkginfo ~= 1.12.0
```

Comment:
I prefer indirect deps to go into a dedicated constraint file… Though, since there are precedents of having them here, this request is out of the scope of this PR. It's something to address separately.

Reply:
Yeah, I feel bad about this being here. The "good" news is that I can revert it pretty much immediately once it's locked, since `pip` won't downgrade it. But I don't know if that's better or worse 😅